bitrise.yml

```diff
@@ -426,6 +426,18 @@ workflows:
     - swiftlint-extended@1:
         inputs:
         - linting_path: "$BITRISE_SOURCE_DIR"
+    - deploy-to-bitrise-io@1.9: {}
+    - script@1:
+        inputs:
+        - content: |-
+            set -e
```

Comment:
does bitrise automatically handle inserting `#!/usr/bin/env bash` here? i've always added it, but i'm not sure what bitrise considers best practice here.

Reply:
oh no, it is not inserted automatically. Not sure either what bitrise consider a good practice. 
We should have the same in all scripts step. Let me add a ticket to define a correct format and apply it to all the scripts we have in bitrise.yml